_posts/2021-04-26-changelog-146.markdown

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #146"
+date: 2021-04-26 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6992 -->
+* Provide initial support for [custom derive](https://doc.rust-lang.org/reference/procedural-macros.html#derive-macros)
+  procedural macros. Now the plugin can expand such procedural macro calls and take into account expanded `impl` items
+  in type inference and name resolution (other types of items are ignored).
+
+  Note, it’s only an initial implementation,
+  so it may work in an unexpected way in some cases. The feature is disabled by default for now.To turn it on, you
+  should enable `Use experimental name resolution engine` option in `Preferences | Languages & Frameworks | Rust`
+  settings and enable `org.rust.cargo.evaluate.build.scripts` and `org.rust.macros.proc` experimental features.
+  Don’t forget to reload a project structure after enabling the corresponding features via `Refresh Cargo Projects`
+  action on Cargo tool window. See [tracking issue](https://github.com/intellij-rust/intellij-rust/issues/6908) for
+  more details and the current status of procedural macros support
+
+TODO gif
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7093 -->
+* Now the plugin takes [Cargo config](https://doc.rust-lang.org/cargo/reference/config.html) into account during
+  evaluation of `cfg` conditions. Note, it works since Rust 1.52.0
+
+TODO gif
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6581 -->
+* Introduce navigation bar breadcrumbs. (by [@Kobzol])
```

Comment:
I've disabled this feature because of https://github.com/intellij-rust/intellij-rust/issues/7153 so let's drop this items